easybuild/easyconfigs/b/Bismark/Bismark-0.23.1-foss-2021b.eb

```diff
@@ -0,0 +1,41 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2013-2014 The Cyprus Institute
+# Authors:: Thekla Loizou <t.loizou@cyi.ac.cy>
+# License:: MIT/GPL
+#
+# Updated to 0.23.1
+# J. Sassmannshausen NHS/GSTT
+
+easyblock = 'Tarball'
+
+name = 'Bismark'
+version = '0.23.1'
+
+homepage = 'https://www.bioinformatics.babraham.ac.uk/projects/bismark/'
+description = "A tool to map bisulfite converted sequence reads and determine cytosine methylation states"
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+source_urls = ['https://github.com/FelixKrueger/Bismark/archive/refs/tags/']
+sources = ['%(version)s.tar.gz']
+checksums = ['2ceb620cfc3f1c869f4bda656ab01c242f1da7ec01615b0d23b28c702147a337']
+
+dependencies = [
+    ('Perl', '5.34.0'),
+    ('Bowtie2', '2.4.4'),
+    ('SAMtools', '1.14'),
+]
+
+sanity_check_commands = [ 'bismark --help']
```

Comment:
```suggestion
sanity_check_commands = ['bismark --help']
```

Reply:
I need to pick up my computer glasses!
Thanks for spotting that, that slipped through on my side.